src/components/data/EntitiesContent.js

```diff
@@ -345,9 +345,11 @@ const EntitiesContent = ({
           onClick: () => setDeletingEntities(true)
         }, ['Delete Data'])
       ])
-    }, [h(Link, { style: { marginRight: '1rem' } }, [
-      icon('ellipsis-v-circle', { size: 24 })
-    ])])
+    }, [
+      h(Link, { style: { marginRight: '1rem' } }, [
+        icon('ellipsis-v-circle', { size: 24, 'aria-label': 'selection menu' })
```

Comment:
#2525 will set `aria-hidden` to false automatically if `aria-label` is present. No need to add that here and then remove it once branch is merged...